### PR TITLE
Take motion configuration variables, some layout-fixes

### PR DESCRIPTION
**Pull request by CatoTH**
- Config variables for line length, default line numbering mode and default change recommendation mode are now evaluated.
- If no changes exist, the original view is displayed.
- Fix Dialog./.TinyMCE-Problems with the scroll bars.

### CODE REVIEW

**Comment by CatoTH** [comment]
@emanuelschuetze the two points should be solved now.

File: client/src/app/site/motions/services/motion-repository.service.ts [hunk 1]
```diff
@@ -43,6 +44,16 @@ import { Mediafile } from 'app/shared/models/mediafiles/mediafile';
     providedIn: 'root'
 })
 export class MotionRepositoryService extends BaseRepository<ViewMotion, Motion> {
+
+    // The line length; comes from the config variable motions_line_length
+    private lineLength = 80;
```
Comment (emanuelschuetze) [inline client/src/app/site/motions/services/motion-repository.service.ts]: The default value of [motions_line_length](https://github.com/OpenSlides/OpenSlides/blob/master/openslides/motions/config_variables.py#L84) is 90! 
@CatoTH Should we change it here? Maybe a better way: you can get the default value of the config instead hard coding?

File: client/src/app/site/motions/components/motion-detail-diff/motion-detail-diff.component.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { AfterViewInit, Component, ElementRef, EventEmitter, Input, Output } from '@angular/core';
-import { ViewMotion } from '../../models/view-motion';
+import {LineNumberingMode, ViewMotion} from '../../models/view-motion';
```
Comment (emanuelschuetze) [inline client/src/app/site/motions/components/motion-detail-diff/motion-detail-diff.component.ts]: Add whitespace

File: client/src/app/site/motions/services/motion-repository.service.ts [hunk 3]
```diff
@@ -28,6 +28,7 @@ import { ViewMotionAmendedParagraph } from '../models/view-motion-amended-paragr
 import { CreateMotion } from '../models/create-motion';
 import { MotionBlock } from 'app/shared/models/motions/motion-block';
 import { Mediafile } from 'app/shared/models/mediafiles/mediafile';
+import {ConfigService} from "../../../core/services/config.service";
```
Comment (emanuelschuetze) [inline client/src/app/site/motions/services/motion-repository.service.ts]: Add whitespace

**Comment by CatoTH** [comment]
Ok, done,

**Pull request merged**